src/service/ie-intrinsic-bug.js

```diff
@@ -27,6 +27,10 @@ import {transparentPng} from '../utils/img';
  * @package
  */
 export function ieIntrinsicCheckAndFix(win, opt_platform) {
+  if (IS_ESM) {
```

Comment:
This one is already handled. See https://github.com/ampproject/amphtml/blob/eca5174f4c41ef00ceb9ec4a18b3282aec608fc8/src/service/resources-impl.js#L362-L364

Reply:
Oops!